### PR TITLE
feat(docs): update docs and examples with new run command

### PR DESCRIPTION
**Pull request by schoren**
This PR updates the documentation, examples, and other doc files to use the new `tracetest run` command

## Changes

-

## Fixes

-

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [x] updated the docs
- [ ] added a test

### CODE REVIEW

File: examples/quick-start-github-actions/.github/workflows/start-and-test-on-main.yaml [hunk 1]
```diff
@@ -28,7 +28,7 @@ jobs:
       run: |
         tracetest run test -f ./tracetest/tests/test-api.yaml
         tracetest run test -f ./tracetest/tests/test-api-and-av.yaml
-        tracetest run test -f ./tracetest/tests/transaction-api.yaml
+        tracetest transaction test -f ./tracetest/tests/transaction-api.yaml
```
Comment (kdhamric) [inline examples/quick-start-github-actions/.github/workflows/start-and-test-on-main.yaml:31]: @schoren is this correct??

**Review by kdhamric — APPROVED** [review]

File: docs/docs/cli/configuring-your-cli.md [hunk 2]
```diff
@@ -1,66 +1,62 @@
...
-
-- `your-tracetest-server-url` - The URL to the running Tracetest server you wish to execute the test on. Example: `http://localhost:11633/`
-- `file-path` - The path to the saved Tracetest test. Example: `./mytest.yaml`
-
-```bash wordWrap=true
-docker run --rm -it -v$(pwd):$(pwd) -w $(pwd) --network host --entrypoint tracetest kubeshop/tracetest:latest -s <your-tracetest-server-url> test run  --definition <file-path> --wait-for-result
-```
+# Configuring your CLI
+
+Our web interface makes it easier to visualize your traces and add assertions, but sometimes a CLI is needed for automation. The CLI was developed for users creating tests and executing them each time a change is made in the system, so Tracetest can detect regressions and check service Service Level Objectives (SLOs).
```
Comment (jfermi) [inline docs/docs/cli/configuring-your-cli.md:3]:
```suggestion
Our web interface makes it easier to visualize your traces and add assertions, but sometimes a CLI is needed for automation. The CLI was developed for users creating tests and executing them each time a change is made in the system, so Tracetest can detect regressions and check your Service Level Objectives (SLOs).
```

File: docs/docs/cli/configuring-your-cli.md [hunk 3]
```diff
@@ -1,66 +1,62 @@
...
+
+Every time the test is run, changes are detected and, if any change is introduced, we use Tractest's [versioning](../concepts/versioning) mechanism to ensure that it will not cause problems with previous test runs.
+
+**How to Use**:
+
+```sh
+tracetest run test --file <file-path>
+```
+
+### Running Tracetest CLI From Docker
```
Comment (jfermi) [inline docs/docs/cli/configuring-your-cli.md:49]:
```suggestion
### Running Tracetest CLI from Docker
```

**Review by jfermi — COMMENTED** [review]
A couple of editing suggestions.

**Pull request merged**